src/RazorSdk/SourceGenerators/RazorSourceGenerationOptions.cs

```diff
@@ -14,19 +14,10 @@ internal class RazorSourceGenerationOptions : IEquatable<RazorSourceGenerationOp
 
         public RazorConfiguration Configuration { get; set; } = RazorConfiguration.Default;
 
-        /// <summary>
-        /// Gets a flag that determines if the source generator waits for the debugger to attach.
-        /// <para>
-        /// To configure this using MSBuild, use the <c>_RazorSourceGeneratorDebug</c> property.
-        /// For instance <c>dotnet msbuild /p:_RazorSourceGeneratorDebug=true</c>
-        /// </para>
-        /// </summary>
-        public bool WaitForDebugger { get; set; } = false;
```

Comment:
Any motivation for removing the debug flag?

Reply:
We stopped using it when we migrated to the incremental APIs. You kinda need to break inside one of the steps for this to work reliably and we never did that.